scripts/circle-release-next-packages.sh

```diff
@@ -48,7 +51,15 @@ else
 
 
   echo $'\nBuilding packages'
-  yarn packages:build
+
+  for PACKAGE in ui data toolkit runtime
+  do
+    start=$(date +%s%N)
+    yarn workspace @grafana/$PACKAGE run build
+    runtime=$((($(date +%s%N) - start)/1000000))
+
+    exit_if_fail ./ci-metrics-publisher.sh "grafana.ci-buildtimes.$CIRCLE_JOB.$PACKAGE=$runtime"
```

Comment:
Might want to make sure you just publish this on the [master branch](https://github.com/grafana/grafana/blob/11304b14b63fe25407a5f3bd3cf1808a3a95f6ce/scripts/ci-job-succeeded.sh#L13).